projects/ccf-shared/src/lib/components/call-to-action/call-to-action.component.scss

```diff
@@ -0,0 +1,71 @@
+.content{
```

Comment:
Move all styles into a top level `:host` selector. Move `.content` styles into the top level `:host`.

Reply:
Removing the content class significantly changes the borders of the dialog box.